packages/common/src/adapters/audioTransactions.ts

```diff
@@ -0,0 +1,68 @@
+import { full } from '@audius/sdk'
+
+import {
+  TransactionDetails,
+  TransactionMethod,
+  TransactionType
+} from '~/store/ui/transaction-details/types'
+
+export const audioTransactioFromSdk = (
```

Comment:
lol just noticed this 
```suggestion
export const audioTransactionFromSdk = (
```